DigitalLearningSolutions.Data.Tests/Services/ActionPlanServiceTests.cs

```diff
@@ -145,7 +145,8 @@ public void AddResourceToActionPlan_calls_expected_insert_data_service_methods()
         }
 
         [Test]
-        public async Task GetIncompleteActionPlanResources_returns_empty_list_if_no_incomplete_learning_log_items_found()
+        public async Task
+            GetIncompleteActionPlanResources_returns_empty_list_if_no_incomplete_learning_log_items_found()
         {
             // Given
             const int delegateId = 1;

```

Comment:
If the responsibility is the caller now, do we need the line below this mocking the config?

Reply:
Good spot, removed it.